layers/opentelemetry/Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG PHP_VERSION
+ARG BREF_VERSION
+FROM bref/build-php-$PHP_VERSION:$BREF_VERSION AS ext
+
+RUN MAKEFLAGS="-j $(nproc)" && pecl install opentelemetry-1.0.0 > /dev/null
```

Comment:
The ` &&` is a typo?

Reply:
This layer is based off the grpc layer which does the same [grpc/Dockerfile#L5](https://github.com/brefphp/extra-php-extensions/blob/master/layers/grpc/Dockerfile#L5 ) but now you mention, setting makeflags may be entirely unnecessary. Certainly examples of pecl installs for other extensions do not appear to need makeflags.

``MAKEFLAGS="-j $(nproc)" &&``

Reply:
I ran this without the `&&` and it worked fine. The grpc one is typoed, too.

Reply:
Cool, I dropped the makeflags altogether, they don't seem to be necessary